stack-8.10.1.yaml

```diff
@@ -5,10 +5,10 @@ packages:
 - ./ghcide/
 
 extra-deps:
-- Cabal-3.0.2.0
 # - cabal-helper-1.1.0.0
 - github: DanielG/cabal-helper
   commit: 79a5608778493bf32e74b54bbf1ea2729941e50f
+- Cabal-3.0.2.0
```

Comment:
Why do we keep it here but remove it elsewhere?

Reply:
This stackage snapshot contains to Cabal-3.2, others already contain Cabal-3.0.1, and stylish-haskell requires Cabal < 3.1